lnwallet/channel.go

```diff
@@ -2460,6 +2460,46 @@ func (lc *LightningChannel) evaluateHTLCView(view *htlcView, ourBalance,
 	skipUs := make(map[uint64]struct{})
 	skipThem := make(map[uint64]struct{})
 
+	// fetchParentEntry is a helper method that will fetch the parent of
+	// entry from the corresponding update log.
+	fetchParentEntry := func(entry *PaymentDescriptor,
+		remoteLog bool) *PaymentDescriptor {
+
+		var (
+			updateLog *updateLog
+			logName   string
+		)
+
+		if remoteLog {
```

Comment:
Can be moved inside the case statement

Reply:
What can be moved exactly?

Reply:
`updateLog` and `logName` are only used inside the switch case

Reply:
Update log is used on line 2481, felt right to do the log name in the same if

Reply:
ah, ok